### DISCUSSION TITLE
Crazyflie Swarm

### DISCUSSION THREAD

**Discussion by PumaRub** [bitcraze/discussions#163] [Q&A]
Hello together,

I am trying to fly two drones at the same time. For this I use the following code from Bitcraze:  https://pastebin.com/ekBT31ZS.  I changed it a bit and commented out some parts because I use only two drones instead of 3.

First question: Did I do everything right?
Second question: My drones do not even take off normally, as the specified altitude. I'm using a Loco Pos System.
With other scripts, however, I can use the Crazyflies to fly to the desired positions. Not with this one.

Thanks in advance,

PumaRub

**Comment by knmcguire** [comment]
Hi! 

It seems that you are using the script right, so not sure if that is it. 

Could you perhaps use try simpler script like [this one](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/swarm/hl-commander-swarm.py) and just remove all the go_to such that the crazyflies will simply take off and land after a few seconds.

The loco positioning system has a tendency to place the z-axis estimate lower than the crazyflie is, which can happen with the setup of the nodes. You can check that in the cfclient by looking at the stateEstimate fields on the flight tab. Eventhough, they fly normally individually, it is always good to double check that.

**Reply by PumaRub** [reply]
Thank you for your reply,

I will try that out in any case. One more question: Can I also address the drones individually with this script? If so, with which command can I tell the drones individually where to fly?

Thank you :)

**Reply by knmcguire** [reply]
ah no, they all will do the same with that script. 

There is no high level commander version of the 'each crazyflie is doing something different in the swarm', but [this script](https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/swarm/swarmSequence.py) is a bit more basic commander send_setpoints, but it shows how you can give some arguments to the function, that you can specify per uri aka per crazyflie
